app/controllers/forgot_password_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ForgotPasswordController < ApplicationController
+  skip_before_action :authenticate_request
+
+  def forgot
+    user = User.find_by(email: params[:_json])
+    if user.present?
+      user.send_password_reset
+      render json: {message: 'We have sent you a password reset email'}, status: :ok
```

Comment:
Please change the text to "We've sent the confirmation code to your email address".